website/src/components/_Feedback/index.tsx

```diff
@@ -1,24 +0,0 @@
-import React from 'react'
```

Comment:
do you still need them in the future?
![image](https://user-images.githubusercontent.com/38236089/192229449-15df5402-615c-4869-a849-97f1dc4fef7a.png)